test/unit/tracks/text-track-settings.test.js

```diff
@@ -9,6 +9,16 @@ const tracks = [{
   label: 'test'
 }];
 
+const defaultSettings = {
```

Comment:
We could suggest that vtt.js provides an API which allows user settings, including whether to override styling in the WebVTT or not, and that `Services` and `WebVTTSet` are one thing which drive that flexible API, but if `Services` does not exist, it's still possible to drive the user settings API. That's what I was thinking we should create in our fork, and then submit a PR to the Mozilla repo.

Because inside vtt.js is by far the easiest place to detect whether the WebVTT has styling or not ;-)